assets/style.css

```diff
@@ -23,6 +23,17 @@ html {
   scroll-behavior: smooth;
 }
 
+/* Custom Scrollbar */
```

Comment:
While I appreciate the change and it does look nice, we will not be merging this change.
The reason for this is that scroll bars are part of the browser and not the webpage. As such the look and behaviour of scroll bars changes between different browsers, different versions of the same browser, and different operating systems. It's difficult to control how changes we make will look in each of these multitude of contexts. So it's always best to leave scroll bars to be dealt with outside the control of our own projects. 
I hope you still found the process useful, but I will be closing this PR now. 
Thank you and we hope to see more contributions from you in the future